README.md

```diff
@@ -1,148 +1,91 @@
-# Phoenics
+# Data-driven experiment planning strategies 
 
-[![Build Status](https://travis-ci.com/FlorianHase/phoenics.svg?token=rULvnKYmWdFF3JqQBVVW&branch=master)](https://travis-ci.com/FlorianHase/phoenics)
+This repository offers data-driven experiment planning strategies following principles of Bayesian optimization in combination with Bayesian kernel density estimation to perform cost-effective global optimization on physical experiments and/or resource-intensive computations.
```

Comment:
As part of the Experiment Planners, we offer strategies following principles of Bayesian [...] computations.

Reply:
Done - thank you